llvm/test/Transforms/LoopVectorize/AArch64/force-target-instruction-cost.ll

```diff
@@ -117,11 +117,11 @@ define void @test_iv_cost(ptr %ptr.start, i8 %a, i64 %b) {
 ; CHECK-NEXT:    br i1 [[CMP_N11]], label %[[EXIT_LOOPEXIT]], label %[[VEC_EPILOG_SCALAR_PH]]
```

Comment:
can the test be changed so it doesn't hoist everything out? I think the test now doesn't test what it was originally testing

Reply:
I've added a hint to force vectorization on this loop so it should always calculate the cost now (which is what I think this was originally trying to test)